weave/ops_arrow/list_ops.py

```diff
@@ -540,6 +541,8 @@ def unnest(self):
     if not list_cols:
         return self
 
+    self = self._decode_gql_types()
```

Comment:
When is this needed? Do all ops need to do this? If not, what is special about unnest?

Reply:
i suppose also: why not decode all dict-endocded fields if unnest requires such shape?

Reply:
This is needed for converting to pandas from an arrow array with a dictionary encoded struct field. The only special thing about unnest is that it requires breaking out to pandas. If we could implement it natively in arrow we wouldn't need to do this. Since only the struct fields prevent pandas conversion, we don't need to also convert the string fields